pdf-converter/test.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+const conveter = require('./pdf-to-image');
+
+
+let event = {
+    "bucket": "docex",
+    "contentType": "application/pdf",
+    "crc32c": "XBc/yQ==",
+    "etag": "CLaPiMXMseECEAE=",
+    "generation": "1554214591530934",
+    "id": "docex/demo/hr-outsourcing.pdf/1554214591530934",

```

Comment:
If this is a test, can you refer to a public bucket and documents instead of a private one?

Reply:
We have removed this.